README.md

```diff
@@ -7,4 +7,4 @@ Coming soon.
 
 ---
 
-*A thing by @lachlanjc and @zackshapiro.*
+*A thing by @lachlanjc, @zackshapiro, and @mitulp91.*
```

Comment:
Might end up adding a few more people before the end. Want to make sure everyone gets credit. :)